openadapt/events.py

```diff
@@ -61,16 +77,19 @@ def get_events(
     if meta is not None:
         format_num = (
             lambda num, raw_num: f"{num} of {raw_num} ({(num / raw_num):.2%})"
-        )
+        )  # noqa
```

Comment:
Should we specify which rule is being ignored?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507